common/upgrade/upgrade.go

```diff
@@ -38,6 +38,7 @@ const (
 	FixFailAckPackage = sdk.FixFailAckPackage
 
 	BEP128 = sdk.BEP128 // https://github.com/bnb-chain/BEPs/pull/128 Staking reward distribution upgrade
+	BEP151 = "BEP151"   // https://github.com/bnb-chain/BEPs/pull/151 Dex Prune Upgrade
```

Comment:
The hardfork height is a setting in app.toml, I think we missing the modification there.

Reply:
added

Reply:
Do we need to move it to sdk? 
Moving it to sdk can align the code, however it does not seem to be meaningful.